Manifests/ManifestsApple/com.apple.proxy.http.global.plist

```diff
@@ -351,6 +370,8 @@
 	</array>
 	<key>pfm_title</key>
 	<string>Global HTTP Proxy</string>
+	<key>pfm_tvos_min</key>
+	<string>9.0</string>
 	<key>pfm_unique</key>
 	<true/>
 	<key>pfm_version</key>
```

Comment:
Can we bump the `pfm_version` to `2`?

Reply:
The updates to this manifest include no new preferences, only meta and attribute updates.